docs/scRNAseq_basics/import_exo.md

```diff
@@ -0,0 +1,31 @@
+![](images/yourmove.png){width="500"}
+
+
+Now it's your time to shine ! We are going to put into pratice what we 
+have just seen. By using a more complicated use case, you are going to 
+reproduce the whole scRNAseq analysis with Seurat.  
+
+# Dataset test
+
+The dataset for this analysis will be single cell RNAseq from chick embryos 
+from [de Lima *et al*](https://doi.org/10.1038/s41467-021-24157-x). We are 
+going to analyse only single cell of E10 stage of embryos. You can download 
+the dataset at the GEO accession [GSE166981](https://0-www-ncbi-nlm-nih-gov.brum.beds.ac.uk/geo/query/acc.cgi?acc=GSE166981).
+
+!!! question "What type of data do you retrieve ?"
+    1. Fastq Files
+    3. BAM Files
+    2. Expression matrix  
+    --> Right answer (1)
```

Comment:
You should hide the right answer using

??? note "Answer:"
    The downloaded data are expression matrices (mais pas seulement si c'est le tar...)

Reply:
En fait en cliquant sur le custom on peut télécharger uniquement les fichiers que l'on veut. Donc le format bundle (barcode, feature, matrix) du stade E10 par exemple